tests/gene2experiment-check-optimisation.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
similarly, why isn't this just a single 'check optimistation' script?

Reply:
Mostly because there are two bats scripts :-(. I started with a single of everything.

Reply:
I also stumbled with the fact that the versions of the collections (v1 and v6) are different, so it would need sourcing of each version based on the name of the collection.

Reply:
This repo is a bit messy I find...